src/Simplex/Messaging/Agent/Client.hs

```diff
@@ -581,6 +600,7 @@ resubscribeSMPSession c@AgentClient {smpSubWorkers} tSess =
       withRetryInterval ri $ \_ loop -> do
         pending <- atomically getPending
         forM_ (L.nonEmpty pending) $ \qs -> do
+          waitForUserNetwork c
```

Comment:
add to ntf/xftp? currently it's only in resubscribeSMPSession and runSmpQueueMsgDelivery